content/06.discussion.md

```diff
@@ -54,6 +54,54 @@ In assessing the upper bound on the predictive performance of a deep learning mo
 Study-level variability limits classification performance and can lead to underestimating prediction error if the generalization error is estimated by splitting a single dataset.
 Analyses can incorporate data from multiple labs and experiments to capture between-study variation within the prediction model mitigating some of these issues.
 
+### Uncertainty quantification
+
+Deep learning based solutions for biomedical applications require reliability guarantees and a quantification of uncertainties.
```

Comment:
I think we should probably soften this lightly from "require" to "could substantially benefit from guarantees on the reliability of predictions and a quantification of uncertainty". Deep learning solutions are currently being used, so require is probably too strong.

Reply:
Agreed. Changed the sentence as you suggested.